pkg/volume/iscsi/iscsi_util.go

```diff
@@ -167,7 +171,11 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) error {
 
 	iscsiTransport = extractTransportname(string(out))
 
-	bkpPortal := b.portals
+	bkpPortal := removeDuplicate(b.portals)
```

Comment:
At first I thought if user passes duplicate portal IPs like this, iscsiadm will be called multiple times unnecessarily, but it was my misunderstanding. 
```
    iscsi:
      targetPortal: 192.168.122.85:3260
      portals: ['192.168.122.85:3260', '192.168.122.7:3260']
      iqn: iqn.2017-05.com.example:rhel7
```
Actually waitForPathToExist() checks existing path so even if user specify duplicate portal IPs in the config, duplicate IPs will be skipped.

I'll remove removeDuplicate() logic from attachdisk()

Thanks,